app.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="h-screen flex flex-col font-sans">
+    <div class="h-screen flex flex-col font-sans text-primary-text bg-primary-bg">
```

Comment:
```suggestion
    <div class="h-screen font-sans text-primary-text bg-primary-bg">
```

Reply:
I needed the flex but I don't think the flex-col is needed 👍

Reply:
Wait, I needed the col too. I keep mixing up which is the browser default.